packages/styles/scss/components/card/index.scss

```diff
@@ -44,6 +44,7 @@
       @include carbon--type-style('body-short-02');
 
       margin-bottom: $carbon--spacing-07;
+      flex: 1;
```

Comment:
@ariellalgilmore Looks like this is causing the issue reported by @jeffchew

Reply:
@kennylam @jeffchew fixed!